packages/neuron-ui/src/locales/zh-tw.json

```diff
@@ -376,9 +376,11 @@
         "remove-ckb-data-tip": "請將 CKB Node 數據從 {{prevPath}} 移動至 {{currentPath}}然後點擊 \"數據轉移完成\" 開始從已知區塊同步; 否則點擊 \"重新同步\" 開始重新同步.",
         "remove-indexer-data-tip": "請將 Indexer 數據從 {{prevPath}} 移動至 {{currentPath}}然後點擊 \"數據轉移完成\" 開始從已知區塊同步; 否則點擊 \"重新同步\" 開始重新同步.",
         "cancel": "取消",
-        "resync-describe": "註意: 重新同步不會刪除當前已同步的 CKB Node(Indexer) 數據.",
+        "resync-ckb-node-describe": "註意: 重新同步不會刪除當前已同步的 CKB Node 數據，但會刪除已同步的 Indexer 數據。",
+        "resync-indexer-describe": "註意: 重新同步不會刪除當前已同步的 CKB Node(Indexer) 數據.",
```

Comment:
Description in Chinese mentions `ckb node(indexer) data` while in English it only mentions `Indexer data`